src/ets_lru.app.src

```diff
@@ -13,9 +13,6 @@
 {application, ets_lru, [
     {description, "ETS Base LRU Cache"},
     {vsn, git},
-    {modules, [
```

Comment:
Why are we removing `modules`?

Reply:
As mentioned in a [previous commit to couchdb](https://github.com/apache/couchdb/commit/7be0ab849d57303b93b8cc1d92ce2a23be3c702f) the `modules` application property in .app.src files is unnecessary and error prone to keep in sync with existing module files, and is auto-generated in .app files by rebar.